### ISSUE TITLE
Enable composer cache

### ISSUE THREAD

**Issue by joelhandwell** [haapp/w_apache#81]
composer -h will get following error. composer recipe needs to create .cache directory with proper file permission to enable cache.
```
Cannot create cache directory /var/www/.cache/composer/repo/https---packagist.org/, or directory is not writable. Proceeding without cache
Cannot create cache directory /var/www/.cache/composer/files/, or directory is not writable. Proceeding without cache
```

**Comment by misoman** [comment]
the error appear due non-matching owner cache directory and user who initiate composer. solution: set composer_home variable (in attributes: default['composer']['home_dir']) to folder with same user permission as user who initiate composer. In our case: www-data.

**Comment by joelhandwell** [comment]
Yes, let's set default['composer']['home_dir'] = '/var/www/.composer'. And let serverspec check if the dir exists and have proper permission. Usually unix user home directory is /home/usrname but www-data's home dir is /var/www.
